setup.py

```diff
@@ -12,13 +12,15 @@
     long_description=readme,
     long_description_content_type='text/markdown',
     name="tableau_utilities",
-    version="2.1.21",
+    version="2.1.22",
```

Comment:
@JustinGrilli  We could call this a `2.2.0` release since it adds the subpackage.  But it doesn't really matter so up to you on the versioning.  If you are willing to merge this as is then let's stick with 2.1.22